File: app/src/main/java/by/yahorfralou/plaincalendar/widget/view/configure/IConfigureView.java
package by.yahorfralou.plaincalendar.widget.view.configure;

import java.util.List;

import by.yahorfralou.plaincalendar.widget.model.CalendarBean;
import by.yahorfralou.plaincalendar.widget.model.WidgetBean;

public interface IConfigureView {
    void displayCalendarsDialog(List<CalendarBean> calendars);

    void onWidgetSettingsLoaded(WidgetBean widget);

    void showCalendarsLoadProgress(boolean isShow);

    void showNoWidgets(boolean isShow);

    void onCalendarSettingsSaved();

    void notifyChangesAndFinish();
}

File: app/src/main/java/by/offvanhooijdonk/plaincalendar/widget/ui/configure/IConfigureView.kt
package by.offvanhooijdonk.plaincalendar.widget.ui.configure

import by.offvanhooijdonk.plaincalendar.widget.model.CalendarModel
import by.offvanhooijdonk.plaincalendar.widget.model.WidgetModel

@Deprecated("")
interface IConfigureView {
    fun displayCalendarsDialog(calendars: List<CalendarModel>)
    fun onWidgetSettingsLoaded(widget: WidgetModel)
    fun showCalendarsLoadProgress(isShow: Boolean)
    fun showNoWidgets(isShow: Boolean)

    fun notifyChangesAndFinish()
}
